sled-agent/src/http_entrypoints.rs

```diff
@@ -31,7 +30,6 @@ type SledApiDescription = ApiDescription<SledAgent>;
 pub fn api() -> SledApiDescription {
     fn register_endpoints(api: &mut SledApiDescription) -> Result<(), String> {
         api.register(disk_put)?;
-        api.register(filesystem_put)?;
```

Comment:
Initialization of datasets is done implicitly when requesting services that have attached datasets.